### PR TITLE
fix: disable rollback button for apps with multiple sources

### LINKED ISSUE #11759: 2.6 RC1: clicking the Rollback button in a multi-source app causes UI error
Checklist:

**Describe the bug**

If I click the History & Rollback button in the UI for a multi-source app, I get a UI error. 

**To Reproduce**

Use this app: 

```yaml
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  name: test
  namespace: argocd
spec:
  destination:
    namespace: default
    server: https://kubernetes.default.svc
  project: default
  sources:
  - ref: elastic
    repoURL: https://github.com/crenshaw-dev/values-file-repo
  - helm:
      valueFiles:
      - $elastic/values-prod.yaml
    path: helm-guestbook
    repoURL: https://github.com/argoproj/argocd-example-apps
    targetRevision: HEAD
```

View the app in the UI and click the History & Rollback button.

**Expected behavior**

Since we don't yet fully support UI actions for multi-source apps, I think the button should be disabled, and the tooltip should describe _why_ it's disabled.

**Screenshots**
image

**Version**

2.6.0-rc1

### PR DESCRIPTION
**Pull request by ishitasequeira**
Fixes: #11759  

After changes:
image

Signed-off-by: ishitasequeira <ishiseq29@gmail.com>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: ui/src/app/applications/components/application-details/application-details.tsx [hunk 1]
```diff
@@ -638,9 +639,15 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{app
             },
             {
                 iconClassName: 'fa fa-history',
-                title: <ActionMenuItem actionLabel='History and rollback' />,
-                action: () => this.setRollbackPanelVisible(0),
-                disabled: !app.status.operationState
+                title: (hasMultipleSources ? (<React.Fragment>
+                    <ActionMenuItem actionLabel=' History and rollback' />
+                    {helpTip('Rollback is disabled for apps with multiple sources')}
```
Comment (crenshaw-dev) [inline ui/src/app/applications/components/application-details/application-details.tsx:644]:
```suggestion
                    {helpTip('Rollback is not supported for apps with multiple sources')}
```

**Review by crenshaw-dev — APPROVED** [review]

**Comment by crenshaw-dev** [comment]
@ishitasequeira can you fix UI lint?

**Comment by crenshaw-dev** [comment]
Cherry-picked onto release-2.6 for 2.6.0-rc2.

**Comment by similark** [comment]
Hi!
Is this going to be left disabled for the near future? Is anyone working on enabling this for multi-sources application?

**Comment by ankitabhopatkar13** [comment]
I understand that the `multiple sources` is in the `beta` version.

I don't see it in the `2.7 or 2.8` roadmap either, but still curious if this going to be enabled again anytime soon?

**Pull request merged**